apps/test/unit/code-studio/pd/application_dashboard/summaryTest.js

```diff
@@ -29,17 +30,50 @@ describe("Summary", () => {
       [
         200,
         {"Content-Type": "application/json"},
-        JSON.stringify({"csf_facilitators":{"unreviewed":{"locked":0,"unlocked":1},"pending":{"locked":0,"unlocked":0},"accepted":{"locked":0,"unlocked":0},"declined":{"locked":0,"unlocked":0},"waitlisted":{"locked":0,"unlocked":0}},"csd_facilitators":{"unreviewed":{"locked":0,"unlocked":0},"pending":{"locked":0,"unlocked":0},"accepted":{"locked":0,"unlocked":0},"declined":{"locked":0,"unlocked":0},"waitlisted":{"locked":0,"unlocked":0}},"csp_facilitators":{"unreviewed":{"locked":0,"unlocked":0},"pending":{"locked":0,"unlocked":0},"accepted":{"locked":0,"unlocked":0},"declined":{"locked":0,"unlocked":0},"waitlisted":{"locked":0,"unlocked":0}},"csd_teachers":{"unreviewed":{"locked":0,"unlocked":1},"pending":{"locked":0,"unlocked":0},"accepted":{"locked":0,"unlocked":0},"declined":{"locked":0,"unlocked":0},"waitlisted":{"locked":0,"unlocked":0}},"csp_teachers":{"unreviewed":{"locked":1,"unlocked":1},"pending":{"locked":0,"unlocked":0},"accepted":{"locked":0,"unlocked":0},"declined":{"locked":0,"unlocked":0},"waitlisted":{"locked":0,"unlocked":0}}})
+        JSON.stringify({
+          csf_facilitators: {
+            unreviewed: { locked: 0, unlocked: 1 },
+            pending: { locked: 0, unlocked: 0 },
+            accepted: { locked: 0, unlocked: 0 },
+            declined: { locked: 0, unlocked: 0 },
+            waitlisted: { locked: 0, unlocked: 0 },
+          },
```

Comment:
This is much more readable!
I'm surprised we didn't split up that long line earlier.

Reply:
All credit goes to [Prettier](https://github.com/prettier/prettier)